salt/states/pkg.py

```diff
@@ -1241,10 +1245,14 @@ def installed(
     failed_hold = None
     if targets or to_reinstall:
         reinstall = bool(to_reinstall)
+        force = False
+        if salt.utils.is_freebsd():
+            force = True    # Downgrades need to be forced.
```

Comment:
I was testing with pkg.installed.  You can not install a package when the version is less than the currently installed version without doing a force.